### ISSUE TITLE
Dockerfile missing a $

### ISSUE THREAD

**Issue by fleimgruber** [sharelatex/sharelatex-docker-image#53]
`COPY {baseDir} ...` is missing a `$`, see also line comment by @ConorIA [here](https://github.com/sharelatex/sharelatex-docker-image/commit/6d7993722cbe34d4eff67502f4081d3ba029af45).

**Comment by ConorIA** [comment]
I'm not sure if this is at all related, but even after adding the $ to the line in question, I am still getting a ton of errors when I try to start the image: 

```
sharelatex    | runsv document-updater-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv chat-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv real-time-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv filestore-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv web-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv docstore-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv spelling-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv tags-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv clsi-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv track-changes-sharelatex: fatal: unable to start ./run: file does not exist
sharelatex    | runsv nginx: fatal: unable to start ./run: file does not exist
```

**Comment by fleimgruber** [comment]
I am getting these errors as well.

**Comment by ConorIA** [comment]
I think the issue is that the runit scripts are all titles `run.sh`, but the command is calling `run`. I am not too sure where the run command originates. But adding `.sh` to the end of it would probably resolve the issue. 

EDIT: Trying this: `find . -iname "*sh" -exec rename run.sh run '{}' \;`. Will re-build and check the result.

**Comment by ConorIA** [comment]
That fixed it! I cloned the repo added the $ that this original issue mentioned, then executed:

```
cd sharelatex-docker-image/runit
find . -iname "*sh" -exec rename run.sh run '{}' \;
```

Then rebuilt the image and changed the name of the image in my `docker-compose.yml`. Up and running again.

**Comment by fleimgruber** [comment]
Thanks for sharing! Just for reference, on Debian Jessie I used:

```
cd sharelatex-docker-image/runit
find . -name "*.sh" -exec rename -v 's/\.sh$//i' {} \;
```

LGTM now.

**Comment by fleimgruber** [comment]
It looks as if the two issues were fixed in e029c806984aa6de7d6fc39c12ab4f0c5e92217b and ec24fc28fce1152c315d8c5b4f03410728395399 respectively, so closing.
